src/slacko.ml

```diff
@@ -1075,8 +1133,14 @@ let chat_delete token ts chat =
     | #message_error as res -> res
     | _ -> `Unknown_error
 
+let jsonify_attachments = function
```

Comment:
It would be potentially nicer to just have this operate on attachments directly and use an `Option.map` like function to conditionally dispatch a call to this function.